dir.props

```diff
@@ -230,8 +235,9 @@
     <CoreHostOutputDir>$(BaseOutputRootPath)corehost\</CoreHostOutputDir>
 
     <PackagesBasePath Condition="'$(PackagesBasePath)'==''">$(BinDir)$(OSPlatformConfig)/</PackagesBasePath>
-    <PackagesOutDir Condition="'$(PackagesOutDir)'==''">$(PackagesBasePath)packages/</PackagesOutDir>
-    <SymbolPackagesOutDir Condition="'$(SymbolPackagesOutDir)'==''">$(PackagesOutDir)</SymbolPackagesOutDir>
+    <PackageOutputPath Condition="'$(PackageOutputPath)'==''">$(PackagesBasePath)packages/</PackageOutputPath>
+    <AssetOutputPath Condition="'$(AssetOutputPath)'==''">$(PackageOutputPath)</AssetOutputPath>
+    <SymbolPackageOutputPath Condition="'$(SymbolPackageOutputPath)'==''">$(PackageOutputPath)</SymbolPackageOutputPath>
```

Comment:
Do we want to default these to a assets and symbols sub-dir by default?

Reply:
These being in the same folder as the packages actually means that there are two packages with same ident+version but different contents in the packages dir "feed". Symbol packages are currently spec'd to go in `assets/`. So far, I've just put the symbol packages in the assets dir root. We should figure out where symbol packages should all go because eventually we'll need to gather them up to be able to release a build properly.

Reply:
(Deleted old comment--I severely misread this.)

The isolated build has always had symbol packages in the same dir as lib packages, but it would be more like the other Core repos to have a separate symbol package dir. Leaving it the same as it was seems like a safer bet, though.

Reply:
Right - that was my thinking - leave the defaults alone.  I didn't want to change it here.